app/UpdateStore.tsx

```diff
@@ -7,24 +7,22 @@ import { PageHeader } from "./ui/PageHeader";
 import { FormLabel } from "./ui/FormLabel";
 import { FormGroup } from "./ui/FormGroup";
 import { FormControl } from "./ui/FormControl";
+import { Store } from "./types";
 
 export function UpdateStore(props: {
   onUpdateStoreSuccess: () => void;
   onBackClick: () => void;
-  //   prefilledDisplayName: string;
-  //   prefilledFullName: string;
-  //   prefilledSuburb: string;
+  store: Store;
 }): ReactElement {
   const [errorMessage, setErrorMessage] = useState("");
+  const { onUpdateStoreSuccess, onBackClick, store: storeDetails } = props;
 
-  const [displayName, setdisplayName] = useState("");
-  const [fullName, setFullName] = useState("");
-  const [suburb, setSuburb] = useState("");
-
-  const { onUpdateStoreSuccess: onUpdateStoreClick, onBackClick } = props;
+  const [displayName, setdisplayName] = useState(storeDetails?.displayName);
+  const [fullName, setFullName] = useState(storeDetails?.fullName);
+  const [suburb, setSuburb] = useState(storeDetails?.suburb);
   return (
     <Wrapper>
-      <PageHeader> STORE NAME</PageHeader>
+      <PageHeader>{storeDetails?.displayName}</PageHeader>
```

Comment:
let's remove the rename to `storeDetails` on line 18, and since it's no longer optional the `?` isn't doing anything

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/d967b16ac286cf0316ec4e15e32f4b4150fba93d